services.html

```diff
@@ -197,18 +199,23 @@ <h4>Service Three</h4>
 
         <!-- ========== Service Category 3 ========== -->
 
+        <div style="width: 100%; height: 250px; background-color: lightgrey;">
+          placeholder
+        </div>
+
```

Comment:
@nickfryar Use a placeholder image from [placeholder.com](https://placeholder.com) and center that background image, it'll allow for easier inserting of the image in the upcoming milestone.

Reply:
Good for now, but for the future, it's best to just use a URL referencing placeholder.com instead of downloading the image.

e.g. `<img src="https://via.placeholder.com/150x150">`